app/experimenter/kinto/tasks.py

```diff
@@ -25,13 +29,14 @@ def push_experiment_to_kinto(experiment_id):
     metrics.incr("push_experiment_to_kinto.started")
 
     experiment = Experiment.objects.get(id=experiment_id)
-    ExperimentBucketNamespace.request_namespace_buckets(
-        experiment.normandy_slug,
-        experiment,
-        NIMBUS_DATA["ExperimentDesignPresets"]["empty_aa"]["preset"]["arguments"][
-            "bucketConfig"
-        ]["count"],
-    )
+    if not ExperimentBucketRange.objects.filter(experiment=experiment).exists():
+        ExperimentBucketNamespace.request_namespace_buckets(
```

Comment:
to accomodate experiments that go from accepted -> rejected -> draft -> review

Reply:
fantastic 👏